CHANGES.md

```diff
@@ -6,6 +6,8 @@ Changes
 
 ### Unreleased
 
+* 2023-01-15 - Improvement: Note to assure matching login image/login image text.
+* 2023-01-14 - Improvement: Add Behat test steps for multilanguage static page content.
```

Comment:
This line is not related to this PR and is already there two lines further down in this file. Maybe you didn't apply this commit onto the latest master?